### ISSUE TITLE
Remove start_time attribute from events table

### ISSUE THREAD

**Issue by ChrisBr** [openSUSE/osem#1154]
start time is now in event_schedules table.

@Ana06 ^^

**Comment by Ana06** [comment]
Yes, but I think you can't remove it as it is used here: https://github.com/openSUSE/osem/blob/master/lib/tasks/several_schedules.rake#L14

I think that @hennevogel  told me that you remove attributes in the next release after the tasks has been run.

**Comment by ChrisBr** [comment]
Ok, fine for me!

**Comment by nishanthvijayan** [comment]
room_id needs to be removed as well

**Comment by Ana06** [comment]
Yes, but it has the same problem, it is in the task. We can not remove those attributes until the task is run.
